frontend/ui/uimanager.lua

```diff
@@ -374,6 +378,10 @@ end
 function UIManager:sendEvent(event)
     if #self._window_stack == 0 then return end
 
+    if type(event) == "string" then
```

Comment:
Yes. I found that writing Event:new is redundant.
It looks like the broadcastEvent() and sendEvent() functions do not use event loop.

No matter, I will revert this change, as it won't impact other logic. We can discuss the detail later.